metron-platform/metron-pcap-backend/src/main/java/org/apache/metron/spout/pcap/HDFSWriterCallback.java

```diff
@@ -116,7 +117,11 @@ public HDFSWriterCallback withConfig(HDFSWriterConfig config) {
     public List<Object> apply(List<Object> tuple, EmitContext context) {
         byte[] key = (byte[]) tuple.get(0);
         byte[] value = (byte[]) tuple.get(1);
-        if(!config.getDeserializer().deserializeKeyValue(key, value, KeyValue.key.get(), KeyValue.value.get())) {
```

Comment:
KeyValue as a whole is unused now, right?  Can we just delete the class entirely at this point?